packages/python/plotly/plotly/io/_html.py

```diff
@@ -57,22 +57,10 @@ def to_html(
         is included in the output.  HTML files generated with this option are
         fully self-contained and can be used offline.
 
-        If 'cdn', a script tag that references the plotly.js CDN is included
-        in the output. The url used is versioned to match the bundled plotly.js.
-        HTML files generated with this option are about 3MB smaller than those
```

Comment:
I think we should keep `cdn` option as it could help reduce smaller HTML files.
But for loading that we shouldn't use `requirejs`.

Reply:
Oh ok - but we don't have a cdn currently right?

Reply:
Yes we publish every version to CDN.
It's only the `-latest` version is no longer updated (https://github.com/plotly/plotly.js/pull/5697) since plotly.js v2 and stayed as latest v1 version i.e. `1.58.5`.
So we should be able to continue loading from CDN by specifying the exact version.
To achieve that you could add a separate `script` tag instead of `requirejs` to load e.g.
```html
<script src="https://cdn.plot.ly/plotly-2.35.1.min.js" charset="utf-8"></script>
```
Or alternatively for async loading
```html
<script type="module">
    import "https://cdn.plot.ly/plotly-2.35.2.min.js"
    Plotly.newPlot("gd", [{ y: [1, 2, 3] }])
</script>
```